2-Browsers/Week1/homework/ex5-catWalk/index.js

```diff
@@ -21,8 +22,31 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 
    https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif
 -----------------------------------------------------------------------------*/
+const catGif = document.querySelector('img');
+const catWalkUrl = 'http://www.anniemation.com/clip_art/images/cat-walk.gif';
+const catPlayUrl ='https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif';
+let catLeft = 0;
+let intervalWalk = setInterval(catWalk, 50);
+
 function catWalk() {
-  // TODO complete this function
+  catLeft += 10;
+  catGif.style.top = '30%';
+  catGif.style.left = catLeft.toString() + 'px';
+  const middleScreen = window.screen.width / 2;
+  const catLocation=parseInt(catGif.style.left);
```

Comment:
```suggestion
  const catLocation = parseInt(catGif.style.left);
```
Space missing here :)

Reply:
done :)